staging/src/k8s.io/sample-cli-plugin/hack/sample-plugin-template.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: sample-plugin
```

Comment:
Change that to `ns`, since the sample-plugin uses that name. Or change the `ns` everywhere in the sample-plugin code. Currently this is inconsistent.

Reply:
Yes, totally makes more sense!